.github/s2i-native-images.yaml

```diff
@@ -4,9 +4,13 @@ buildScript: .github/build-s2i-native-images.sh
 versions:
   - 20.2.0-java11
   - 20.3.0-java11
+  - 20.3.1-java11
+  - 21.0.0-java11
 tags:
   - id: 20.2-java11
     target: 20.2.0-java11
   - id: 20.3-java11
-    target: 20.3.0-java11    
-versionCheck: false
+    target: 20.3.1-java11    
+  - id: 21.0-java11
+    target: 21.0.0-java11    
+versionCheck: true
```

Comment:
I see you changed the value of this. What does it do?

Reply:
Actually, I should not have changed.

This is triggering the test verifying the graalvm version installed by running the container `--version`.